### PR TITLE
wol: Homepage inclusion

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Added `homepage` key to `package.yml` (Part of #411)
- Replaced old source with new one

**Test Plan**

- Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/w/wol/package.yml [hunk 1]
```diff
@@ -12,7 +13,7 @@ description: |
     provides SecureON which extends wake-on-lan with a password. This feature
     is also provided by wol.
 setup      : |
-    %configure
+    ./configure $(sed 's|--runstatedir=/run||g' <<< "%CONFOPTS%")
```
Comment (TraceyC77) [inline packages/w/wol/package.yml:16]: My apologies if this would be obvious to someone else, but why was this change made?

**Review by TraceyC77 — CHANGES_REQUESTED** [review]
This PR does not just add a home page. It also changes the configure script for the build. Please do some basic testing to ensure the packaged worked as before with the new configopts, and add it to your test plan.

File: packages/w/wol/package.yml (hunk 1 above)
Comment (joebonrichie) [reply packages/w/wol/package.yml:16]: Old configure scripts don't support runstatedir which is part of our default confopts

Comment (malfisya) [reply packages/w/wol/package.yml:16]: This one ^. `%configure` and `%reconfigure` doesn't work. So i just followed this trick [here](https://github.com/getsolus/help-center-docs/issues/411)

**Comment by malfisya** [comment]
whoops, got accidentallly deleted. reopening. will try add test plan if still necessary

**Review by TraceyC77 — APPROVED** [review]
Thanks for explaining the build change. This looks good to go.

**Pull request merged**